proxy/handler.go

```diff
@@ -0,0 +1,52 @@
+package proxy
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/ONSdigital/dp-legacy-cache-proxy/config"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+func (proxy *Proxy) Handler(ctx context.Context, w http.ResponseWriter, req *http.Request, cfg *config.Config) {
```

Comment:
It looks like the only thing that the config is being used for is to get the Babbage URL, so you can simply pass that instead.